src/helpers/Constants.js

```diff
@@ -40,6 +40,8 @@ export let screenWidth = width;
 export let screenHeight = height;
 export let isSmallScreen = screenWidth <= 340;
 export let isShortScreen = screenHeight <= 600;
+export const ratio = screenWidth < screenHeight ? screenHeight / screenWidth : screenWidth / screenHeight;
+export const isTablet = Platform.isPad || (ratio < 1.6 && Math.max(screenWidth, screenHeight) >= 900);
```

Comment:
Where did you get the 1.6 ratio limit?

Reply:
It was a guesstimation in a Stack-Overflow answer.

Reply:
Did you test it?

Reply:
Yes, it worked on the devices I've tested; and I've added the screen size for good measure

Reply:
Cool